pennylane/ops/op_math/__init__.py

```diff
@@ -64,6 +64,9 @@
     ~ControlledQubitUnitary
     ~CY
     ~CZ
+    ~CH
+    ~CNOT
+    ~Toffoli
```

Comment:
any reason why you didn't register all of them here (eg. CCZ or CSWAP)?

Reply:
Ooooops